### PR TITLE
[don't merge] Add jumpstart swiss and elimination tournament formats.

### PR DESCRIPTION
**Pull request by hitch17**
Adds JumpstartEliminationTournament and JumpstartSwissTournament.

I'll update the file once the full set is spoiled and the contents of the jumpstart packs are public.

### CODE REVIEW

File: Mage.Server/config/config.xml [hunk 1]
```diff
@@ -104,6 +104,8 @@
         <tournamentType name="Sealed Elimination (Cube)" jar="mage-tournament-sealed.jar" className="mage.tournament.SealedEliminationTournament" typeName="mage.tournament.SealedEliminationCubeTournamentType"/>
         <tournamentType name="Sealed Swiss" jar="mage-tournament-sealed.jar" className="mage.tournament.SealedSwissTournament" typeName="mage.tournament.SealedSwissTournamentType"/>
         <tournamentType name="Sealed Swiss (Cube)" jar="mage-tournament-sealed.jar" className="mage.tournament.SealedSwissTournament" typeName="mage.tournament.SealedSwissCubeTournamentType"/>
+        <tournamentType name="Jumpstart Elimination" jar="mage-tournament-sealed.jar" className="mage.tournament.JumpstartEliminationTournament" typeName="mage.tournament.JumpstartEliminationTournamentType"/>
+        <tournamentType name="Jumpstart Swiss" jar="mage-tournament-sealed.jar" className="mage.tournament.JumpstartSwissTournament" typeName="mage.tournament.JumpstartSwissTournamentType"/>
```
Comment (JayDi85) [inline Mage.Server/config/config.xml:108]: There are two configs to edit (release and dev testing):

shot_200619_113110

File: Mage/src/main/java/mage/game/jumpstart/JumpstartPoolGenerator.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import com.google.common.io.CharSource;
+import com.google.common.io.Resources;
+
+import mage.cards.Card;
+import mage.cards.decks.Deck;
+import mage.cards.decks.DeckCardInfo;
+import mage.cards.decks.DeckCardLists;
+import mage.game.GameException;
+
+public class JumpstartPoolGenerator {
```
Comment (JayDi85) [inline Mage/src/main/java/mage/game/jumpstart/JumpstartPoolGenerator.java:21]: Miss comments about format and data source (like wiki page with description, source for packs list, etc)

File: Mage.Server/config/config.xml (hunk 1 above)
Comment (hitch17) [reply Mage.Server/config/config.xml:108]: 👍

File: Mage/src/main/java/mage/game/jumpstart/JumpstartPoolGenerator.java (hunk 2 above)
Comment (hitch17) [reply Mage/src/main/java/mage/game/jumpstart/JumpstartPoolGenerator.java:21]: Added a description and links about the format.  I'll update the file with the details of the packs once they're released.

File: Mage.Server/release/config/config.xml [hunk 3]
```diff
@@ -98,6 +98,8 @@
         <tournamentType name="Sealed Elimination (Cube)" jar="mage-tournament-sealed-${project.version}.jar" className="mage.tournament.SealedEliminationTournament" typeName="mage.tournament.SealedEliminationCubeTournamentType"/>
         <tournamentType name="Sealed Swiss" jar="mage-tournament-sealed-${project.version}.jar" className="mage.tournament.SealedSwissTournament" typeName="mage.tournament.SealedSwissTournamentType"/>
         <tournamentType name="Sealed Swiss (Cube)" jar="mage-tournament-sealed-${project.version}.jar" className="mage.tournament.SealedSwissTournament" typeName="mage.tournament.SealedSwissCubeTournamentType"/>
+        <tournamentType name="Jumpstart Elimination" jar="mage-tournament-sealed.jar" className="mage.tournament.JumpstartEliminationTournament" typeName="mage.tournament.JumpstartEliminationTournamentType"/>
+        <tournamentType name="Jumpstart Swiss" jar="mage-tournament-sealed.jar" className="mage.tournament.JumpstartSwissTournament" typeName="mage.tournament.JumpstartSwissTournamentType"/>
```
Comment (JayDi85) [inline Mage.Server/release/config/config.xml:102]: Missing `${project.version}` in that config.

**Comment by JayDi85** [comment]
I'll merge it after new release (after jumpstart set completed #6656).

**Comment by hitch17** [comment]
Sounds good, thanks.

**Pull request merged**